carriers/dhl.js

```diff
@@ -14,6 +14,7 @@ const SHIPPED_TRACKING_DESCRIPTIONS = ['ARRIVAL DESTINATION DHL ECOMMERCE FACILI
 const geography = require('../util/geography');
 
 function DHL(options) {
+    const dhlEcommerceSolutions = new DhlEcommerceSolutions(options && options.dhlEcommerceSolutions);
     const usps = new USPS(options && options.usps);
 
     this.isTrackingNumberValid = function(trackingNumber) {
```

Comment:
I apparently cannot make a comment on L24. This is for L24. Why can't this be on L24???

There are still all the `420...` tracking numbers that might come from a barcode scanner which wouldn't get picked up here, but this behavior is no different than the DHL carrier was before. Also, for DHLeCS, which I think most will be, we do pick those up on L28 and then try the UTAPI logic first. I think this is the right approach, but if we see things assigned to USPS instead of DHL, this is one of the first places to check.